infra/resources/modules/commons/containers.tf

```diff
@@ -89,6 +89,23 @@ resource "azurerm_cosmosdb_sql_container" "activation" {
         order = "Ascending"
       }
     }
+
+    composite_index {
+      index {
+        path  = "/trialId"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/type"
+        order = "Ascending"
+      }
```

Comment:
**suggestion (non-blocking)**: I think the type field is useless in this index
```suggestion
```

Reply:
I made some tests, and with the `type` property, the query consumes `2.83 RU` while without the `type` parameter in the index, the very same query consumes `2.93 RU`.

It's a small difference, but I'd keep the `type` within the index

Reply:
**praise**: 👏 Well done for checking!